static/app/actionCreators/tags.tsx

```diff
@@ -192,3 +208,214 @@ export function fetchSpanFieldValues({
     query,
   });
 }
+
+type FetchOrganizationTagsParams = {
+  orgSlug: string;
+  // TODO: Change this to Dataset type once IssuePlatform is added
+  dataset?: string;
```

Comment:
I initially made the type of this dataset parameter [Dataset](https://github.com/getsentry/sentry/blob/f66b6a9013dc827900f1f6fd087e51cfff06ce43/static/app/views/alerts/rules/metric/types.tsx#L29). However, we would need to add `IssuePlatform='search_issues'`, and this causes type errors elsewhere. Since there is only one place using this hook, I think it's reasonable to leave it as a String now and refactor it later in a different PR.

Reply:
Glad you called this out, this is a good followup after this gets merged.